### PR TITLE
Replace mdc circular progress with mwc

### PR DESCRIPTION
**Pull request by bramkragten**
## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/lovelace/editor/lovelace-editor/hui-dialog-edit-lovelace.ts [hunk 1]
```diff
@@ -76,10 +76,13 @@ export class HuiDialogEditLovelace extends LitElement {
...
-              ?active="${this._saving}"
-              alt="Saving"
-            ></ha-circular-progress>
+            ${this._saving
+              ? html`<ha-circular-progress
+                  active
+                  size="small"
+                  title="Saving"
+                ></ha-circular-progress>`
+              : ""}
```
Comment (ludeeus) [inline src/panels/lovelace/editor/lovelace-editor/hui-dialog-edit-lovelace.ts:85]: Should not the spinner replace the text when active?
image

Comment (bramkragten) [reply src/panels/lovelace/editor/lovelace-editor/hui-dialog-edit-lovelace.ts:85]: I intentionally didn't change the current behavior... But we should align it some time I think. We use different ways at different places

**Comment by bramkragten** [comment]
I'll also bump mwc in this PR, as they just released it and I can drop a hack.

File: src/panels/lovelace/editor/lovelace-editor/hui-dialog-edit-lovelace.ts (hunk 1 above)
Comment (ludeeus) [reply src/panels/lovelace/editor/lovelace-editor/hui-dialog-edit-lovelace.ts:85]: Oh... right, that's not related to this change, just noticed while testing it :D

**Review by ludeeus — APPROVED** [review]

**Pull request merged**